climakitae/warming_levels.py

```diff
@@ -3,6 +3,7 @@
 import xarray as xr
 import holoviews as hv
 from holoviews import opts
+import bokeh.palettes
```

Comment:
when is this package used? If it isn't used, please delete!

Reply:
Sorry - this was a holdover from doing this manually, but not needed for this version! Will delete!